backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/database/Database.kt

```diff
@@ -51,12 +51,10 @@ class Database {
     }
 
     val db by lazy {
-        val ds = PGDataSource()
-        ds.serverName = System.getProperty("app.postgres.host")
-        ds.databaseName = System.getProperty("app.postgres.database")
-        ds.user = System.getProperty("app.postgres.user")
-        ds.password = System.getProperty("app.postgres.password")
-
-        connect(ds)
+        val connectionUrl = "jdbc:postgresql://" +
```

Comment:
what does reWriteBatchedInserts do? Do we require it?

Reply:
It tries to batch following inserts together to make the inserts faster. However that does not seem to work because we have inserts that depend on each other. It was a try, I can remove it again I guess